<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YTDowload</class>
 <widget class="QMainWindow" name="YTDowload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>YT-Download</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>git.ico</normaloff>git.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0
);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>610</width>
     <height>400</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(22, 27,34 );</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(13,17,23);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btnMP4">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>280</y>
         <width>221</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {	
	background-color: rgb(35, 134,54 );
	border: 2px solid rgb(55, 145,74 );
    color:rgb(255, 255,255 );
	border-radius: 16px;
}
QPushButton:hover {	
	background-color: rgb(46, 160, 67);
	border: 2px solid rgb(55, 145,74 );
}
QPushButton:pressed {	
	background-color: rgb(35, 134,54 );
	border: 2px solid rgb(55, 145,74 );
	color:rgb(255, 255,255 );
}</string>
       </property>
       <property name="text">
        <string>Download - MP4</string>
       </property>
      </widget>
      <widget class="QLabel" name="textVersao">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>360</y>
         <width>601</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(96,96,96)</string>
       </property>
       <property name="text">
        <string>Versão 1.01</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="textBuild">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>360</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(96,96,96)</string>
       </property>
       <property name="text">
        <string>github.com/JardeFelicio</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="projeto_Python">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>40</y>
         <width>601</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat SemiBold</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">	color: rgb(255, 255,255 )</string>
       </property>
       <property name="text">
        <string>Projeto Python</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="textResumo">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>521</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(101,101,101)</string>
       </property>
       <property name="text">
        <string>Cole o URL do Youtube e escolha a modalidade de download.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditLink">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>531</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 2px solid rgb(48, 54, 61);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(22, 27, 34);	
	color: rgb(100, 100, 100);
}
QLineEdit:hover {
	border: 2px solid rgb(34, 77,135 );
	background-color: rgb(20, 35, 58);	
}
QLineEdit:focus {
	border: 2px solid rgb(94, 55,255 );	

}</string>
       </property>
       <property name="maxLength">
        <number>150</number>
       </property>
       <property name="placeholderText">
        <string>Cole o URL do Vídeo</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnMP3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>280</y>
         <width>221</width>
         <height>51</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {	
	background-color: rgb(35, 134,54 );
	border: 2px solid rgb(55, 145,74 );
    color:rgb(255, 255,255 );
	border-radius: 16px;
}
QPushButton:hover {	
	background-color: rgb(46, 160, 67);
	border: 2px solid rgb(55, 145,74 );
}
QPushButton:pressed {	
	background-color: rgb(35, 134,54 );
	border: 2px solid rgb(55, 145,74 );
	color:rgb(255, 255,255 );
}</string>
       </property>
       <property name="text">
        <string>Download - MP3</string>
       </property>
      </widget>
      <widget class="QLabel" name="textInfo">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>521</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(101,101,101)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="icone">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>git_white.ico</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="textInfoMotiv">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>521</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat Medium</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(101,101,101)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_close">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>10</y>
         <width>17</width>
         <height>17</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Sair</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;		
	background-color: rgb(255, 0, 0);
}
QPushButton:hover {		
	background-color: rgba(255, 0, 0, 150);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_maximize">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>10</y>
         <width>17</width>
         <height>17</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Minimizar</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;	
	background-color: rgb(85, 255, 127);
}
QPushButton:hover {	
	background-color: rgba(85, 255, 127, 150);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_minimize">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>10</y>
         <width>17</width>
         <height>17</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Maximizar</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;		
	background-color: rgb(255, 170, 0);
}
QPushButton:hover {	
	background-color: rgba(255, 170, 0, 150);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnMP4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
